photon-targeting/src/main/native/include/photon/dataflow/structures/Packet.h

```diff
@@ -21,6 +21,7 @@
 #include <bit>
 #include <string>
 #include <vector>
+#include <cstring>
```

Comment:
was this missing before or just for redundancy?

Reply:
Missing